sass/_grid.scss

```diff
@@ -0,0 +1,58 @@
+/* grid */
+
+@mixin row($inside: false) {
+  @if($inside == true) {
+    margin-left: -$gutter/2;
+    margin-right: -$gutter/2;
+  }
+  @else {
+    width: 100%;
+    max-width: $wrapper;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  &:before, &:after {
+    content: '';
+    display: table;
+  }
+  &:after {
+    clear: both;
+  }
+}
+@mixin col($width: 1, $padding: true) {
+  float: left;
+  width: percentage($width);
+  @if($padding == true) {
+    padding-left: $gutter/2;
+    padding-right: $gutter/2;
+  }
+}
+
+/* Layout */
+.wrapper {
+  @include row();
+}
+.row {
+  @include row();
+}
+.column {
+  @include col(1, false);
+  @include mq(600px) {
+    @include col(1, false);
+  }
+  @include mq(900px) {
+    @include col(1/2, false);
+  }
+}
+.content {
+  @include col;
+  @include mq(900px) {
```

Comment:
You should create variables for your breakpoints. Consider using breakpoint-sass.